db/seeds.rb

```diff
@@ -11,11 +9,787 @@
+DealVenue.create(id: '1040', deal_id: 1016, venue_id: 1008)
+
+Deal.new(id: '1017', title: '$14 for FishHead Curry wotth $26', redeemable: true, multiple_use: true, type_of_deal: 'Discount',
+         description: 'Limited time only', start_date: start_date, expiry_date: end_date,
+         t_c: 'Valid through Burpple only.', pushed: true, merchant_id: 1004, active: true, num_of_redeems: 0,
+         created_at: created_at).save(validate: false)
+DealDay.new(id: '1025', deal_id: 1017, mon: true, tue: true, wed: true, thur: true, fri: true, sat: false, sun: false).save(validate: false)
+DealTime.create(id: '1025', deal_day_id: 1025, started_at: start_time_1, ended_at: end_time_1)
+DealVenue.create(id: '1041', deal_id: 1017, venue_id: 1008)
+
+
+# Seed User Query Table
+UserQuery.create(id: '1000', query: 'lunch', num_count: '1921', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1001', query: 'dinner', num_count: '2895', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1002', query: '1 for 1', num_count: '3921', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1003', query: 'delicious', num_count: '1494', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1004', query: 'cheap', num_count: '2516', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1005', query: 'salmon', num_count: '1891', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1006', query: 'local', num_count: '2019', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1007', query: 'amazing deals', num_count: '1721', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1008', query: 'limited time', num_count: '4912', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1009', query: 'savings', num_count: '3612', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+UserQuery.create(id: '1010', query: 'buffet', num_count: '3812', query_type: 'DealsFeed', created_at: Date.today.beginning_of_day)
+
+# Seed Data for number of wish listers
+i = 1000
+while i < 1132
+  Wish.create(id: i, venue_id: 1000, user_id: i)
+  i = i + 1
+end
+
+while i < 1301
+  Wish.create(id: i, venue_id: 1001, user_id: i)
+  i = i + 1
+end
+
+while i < 1451
+  Wish.create(id: i, venue_id: 1002, user_id: i)
+  i = i + 1
+end
+
+# Seed DealAnalytics for View count for deal 1000
+view_count = 1000
+start_date = DateTime.parse("2015-09-20 00:00:00")
+end_date = DateTime.parse("2015-10-20 00:00:00")
+while start_date <= end_date
+  limit = view_count + rand(20..40)
+  while view_count < limit
+    Viewcount.create(id: view_count, deal_id: 1000, user_id: view_count, created_at: start_date)
```

Comment:
Oh, I didn't seed users yet. But it is not necessary to have actual users so it is ok